### PR TITLE
Create entity for a Fastly Datacenter (aka POP)

### PR DESCRIPTION
**Pull request by nr-jbiggley**
### Relevant information

Parsing Logs-to-Metrics and then synthesizing the metrics to create an entity for a Fastly datacenter (POP)

### Checklist

* [X] I've read the guidelines and understand the acceptance criteria.
* [X] The value of the attribute marked as `identifier` will be unique and valid. 
* [X] I've confirmed that my entity type wasn't already defined. If it is I'm providing an
 explanation above.

### CODE REVIEW

**Review by nr-jbiggley — COMMENTED** [review]
Needed to change the L2M rule name

**Comment by jfjoly** [comment]
@nr-jbiggley the `-` is not accepted in the name of the type. Also the length for the name of the type is limited to 11.
Here is the regexp that it needs to match `/[A-Z][A-Z0-9_]{2,11}/` (more details in the README of this repo). You can use https://regexr.com/ to verify that it's matching the regexp.

File: definitions/ext-fastly-datacenter/definition.yml
Comment (srvaroa) [inline definitions/ext-fastly-datacenter/definition.yml:4]:
```suggestion
type: FASTLY_DATACENTER
```

`-` is a reserved character

File: definitions/ext-fastly-datacenter/definition.yml
Comment (srvaroa) [inline definitions/ext-fastly-datacenter/definition.yml:7]: Some questions: 

- Are `name` and `identifier` meant to be the same (they use `-` and `_`)
- I assume the `fastly-datacenter` property in this telemetry will be considered unique for a given account, is that right?
- Do you know how long are `fastly-datacenter` values? (this is relevant in order to know if we should encode the identifiers, they would remain untouched in the name.

**Review by srvaroa — CHANGES_REQUESTED** [review]
Minor changes and questions, but LGTM.

Thanks @nr-jbiggley !

File: definitions/ext-fastly-datacenter/definition.yml
Comment (nr-jbiggley) [reply definitions/ext-fastly-datacenter/definition.yml:7]: @srvaroa The fastly-datacenter property is unique for a given account. Each Datacenter has a 3 character code that is expressed by Fastly.

**Review by nr-jbiggley — COMMENTED** [review]
Looks solid to me. Thanks for the help @jfjoly & @srvaroa. The first one is always the hardest :)

File: definitions/ext-fastly-datacenter/definition.yml
Comment (srvaroa) [reply definitions/ext-fastly-datacenter/definition.yml]: Fantastic, thanks!

**Comment by srvaroa** [comment]
Sorry @nr-jbiggley I missed this in my previous review: the directory should match exactly this format "$domain-$type" in lowercase.  Can you rename it to `ext-fastly_pop`? I'll merge right after that's done.

**Review by srvaroa — APPROVED** [review]

**Comment by srvaroa** [comment]
thanks @nr-jbiggley !

**Pull request merged**
